public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.styles.ts

```diff
@@ -0,0 +1,104 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { stylesFactory } from '@grafana/ui';
```

Comment:
```suggestion
```

Reply:
Fixed and fixed in `AlertRulesTable.styles.ts` as well